lib/client/js/ui/base/Control.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Base Controller class. This class will be extended in order
+ * to dynamically mount/update/delete UI components from the DOM
+ * @class
```

Comment:
Please include a single line of blank space between descriptions and tags, please.